pxtsim/simdriver.ts

```diff
@@ -706,6 +706,7 @@ namespace pxsim {
                 case 'custom':
                 case 'recorder':
                 case 'extensionsdialog':
```

Comment:
Worth yanking the 'extensionsdialog' case then?

Reply:
removing

Reply:
removed